protocols/upnp/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.1
+- Ignore oneshot sender error when behaviour has dropped.
```

Comment:
```suggestion
- Fix panic caused by oneshot receiver being dropped in the behaviour by checking 
  if the oneshot sender returns an error instead of expecting it not to fail.
```

Thoughts @thomaseizinger

Reply:
That is better, thank you! Instead of going into the implementation details, can we mention _when_ a user would run into this panic? I didn't follow the issue closely but it appears to happen in combination with `Toggle`? Is it when `upnp::Behaviour` gets dropped?

Reply:
Correct, though its more related to calling a function inside of `bool::then_some` than using `bool::then` where it is lazily evaluated (and thus only executing the function if the it the boolean is true). As a result, it causes the behaviour to drop because its not used, which causes the receiver to drop as well and would cause the oneshot sender to return an error (or in our case, it would panic since its expecting it not fail).

Reply:
Okay, so we can say something like:

```
- Fix a panic caused when dropping `upnp::Behaviour` such as when used together with `Toggle`.
```

Reply:
Sounds good to me